website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: Progressive Delivery using Flagger
+hide_title: true
+sidebar_position: 6
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Progressive Delivery using Flagger <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise includes the ability to view [Flagger](https://docs.flagger.app/) [canary](https://docs.flagger.app/usage/how-it-works#canary-resource) objects information in the UI through the Applications > Delivery view.
+
+This guide uses Flux manifests to install Flagger and Linkerd. Using Flux allows us to manage our cluster applications in a declarative way through changes in a Git repository.
+
+## Installing Flagger using Flux
+
+For the Flagger installation, a Kustomization file will be used.
+
+First, add a Namespace resource for Flagger:
+
+```yaml title="flagger/namespace.yaml"
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: flagger
```

Comment:
The Flagger docs talk about installing Flagger in the Linkerd namespace - _maybe_ worth being consistent with this approach.

Reply:
I went for a separate namespace to make the guide more flexible/less opinionated about where things go.

Reply:
there's an inconsistency with the following resources, where they add "add the following manifest to the cluster repository". I'd either remove them and hope that its covered by my suggestions in the pre-req section, or make sure its added to all files

Reply:
Update 👍